packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,147 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+ /* eslint camelcase: [0] */
+
+import '../..';
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+describe(`plugin-calendar`, () => {
+  describe(`#meetings()`, () => {
+
+    let createdMeeting, mccoy, meetingParams, organizer, spark, spock;
+
+    before(`create test users`, () => testUsers.create({
+      count: 3,
+      config: {
+        entitlements: [
+          `spark`,
+          `squaredCallInitiation`,
+          `squaredInviter`,
+          `squaredRoomModeration`,
+          `webExSquared`,
+          `squaredFusionCal`
+        ]
+      }
+    })
+      .then((users) => {
+        [organizer, spock, mccoy] = users;
+
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: organizer.token
+          },
+          config: {
+            device: {
+              preDiscoveryServices: {
+                whistlerServiceUrl: process.env.WHISTLER_API_SERVICE_URL || `http://internal-testing-services.wbx2.com:8084/api/v1`
+              }
+            }
+          }
+        });
+      }));
+
+    const hour = 1000 * 60 * 60;
+    const startInterval = new Date(new Date().getTime() + hour * 2).toISOString();
+    const meetingId = uuid.v4();
+
+    before(`creates a meeting`, () => {
+      meetingParams = {
```

Comment:
Adding a few levels of indirection would the predicates look like this: 

`
      {
        name: `transformMeetingArray`,
        direction: `inbound`,
        test(ctx, response) {
          return Promise.resolve(has(response, `body`));
        },
        extract(response) {
          return Promise.resolve(response.body);
        }
      },
      {
        name: `transformMeeting`,
        direction: `inbound`,
        test(ctx, response) {
          return Promise.resolve(has(response, `body.items`));
        },
        extract(response) {
          return Promise.resolve(response.body.items);
        }
      },
      {
        name: `decryptMeeting`,
        direction: `inbound`,
        test(ctx, response) {
          return Promise.resolve(has(response, `body.items[0]`));
        },
        extract(response) {
          return Promise.resolve(response.body.items);
        }
      },
      {
        name: `decryptPropSubject`,
        direction: `inbound`,
        test(ctx, response) {
          return Promise.resolve(has(response, `body.items[0].encryptedSubject`));
        },
        extract(response) {
          return Promise.resolve(response.body.items);
        }
      }
`

Looked at the example in  plugin-conversation, but was a little unclear. Thoughts @ianwremmel ?